src/map.rs

```diff
@@ -248,7 +251,7 @@ impl Map {
 
         Self::draw_killfeed(canvas, assets, game_state)?;
 
-        Self::draw_debug_lines(canvas, &game_state.debug_lines, camera_position)
+        Ok(())
```

Comment:
Rather than removing debug drawing, we should remove the places that add lines to draw

Reply:
Oops my mistake

Reply:
Fixed